src/server/queries/dashboards.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+var pool = require('../db/index.js');
+
+module.exports = {
+  getOne: function (orgName, repoName) {
```

Comment:
getId() and getLastCommitSha1() seem extraneous now. I think we can just have getOne().